### ISSUE TITLE
Loss Start with 3.9

### ISSUE THREAD

**Issue by FirasAbdennadher** [matterport/Mask_RCNN#2436]
Hi everyone !

Hi, when I train a model on a custom dataset ( 7 classes ) with the same configs and using layers='heads' ,i got a very high loss started with ~3.900.
After 100 epochs, loss became ~1.300 , now am in epochs 180, the loss doesn't decrease around 0.8460.

Epoch 180/200
100/100 [==============================] - 257s - loss: 0.8460 - rpn_class_loss: 0.0319 - rpn_bbox_loss: 0.2793 - mrcnn_class_loss: 0.1216 - mrcnn_bbox_loss: 0.1504 - mrcnn_mask_loss: 0.2628 - val_loss: 1.7520 - val_rpn_class_loss: 0.1110 - val_rpn_bbox_loss: 0.6783 - val_mrcnn_class_loss: 0.3193 - val_mrcnn_bbox_loss: 0.3090 - val_mrcnn_mask_loss: 0.3344
Is that normal loss 0.8 ?
Any good advice? 
thanks.

**Comment by kimile599** [comment]
Hi,

for the high start loss, it is ok. Since you don't share your training steps per epoch. If you are using a large steps, after the first epoch you will have a lower loss. 
For the loss decreasing, check your learning rate if it is too small.

Hope this will help.

**Comment by FirasAbdennadher** [comment]
> Hi,
> 
> for the high start loss, it is ok. Since you don't share your training steps per epoch. If you are using a large steps, after the first epoch you will have a lower loss.
> For the loss decreasing, check your learning rate if it is too small.
> 
> Hope this will help.

Thanks for ur response.
First of all, how do we share training steps per epochs? 
, secondly, i augmented to steps per epochs to 1000 , it really decrease , but the same,
the loss doesn't decrease around 0.8 0.7
(LEARNING_RATE=0.001;LEARNING_MOMENTUM = 0.9 ; WEIGHT_DECAY = 0.0001)

**Comment by kimile599** [comment]
> > Hi,
> > for the high start loss, it is ok. Since you don't share your training steps per epoch. If you are using a large steps, after the first epoch you will have a lower loss.
> > For the loss decreasing, check your learning rate if it is too small.
> > Hope this will help.
> 
> Thanks for ur response.
> First of all, how do we share training steps per epochs?
> , secondly, i augmented to steps per epochs to 1000 , it really decrease , but the same,
> the loss doesn't decrease around 0.8 0.7
> (LEARNING_RATE=0.001;LEARNING_MOMENTUM = 0.9 ; WEIGHT_DECAY = 0.0001)

Then you might change the lr to a lower number and try to train more layers.
